### ISSUE TITLE
Reviewing and publishing atomic rules used in composites

### ISSUE THREAD

**Issue by WilcoFiers** [w3c/wcag-act#459]
Atomic rules that do not directly map to WCAG, but that are used in composite rules don't fit well into how we're currently reviewing rules. We require at least 1 implementation of each rule, the problem is, there is no good way to figure out if such atomic rules are actually implemented.

The issue is that there is no fail-to-fail relationship between those atomic rules and the SC. Something that fails the atomic rule doesn't necessarily fail the SC. The SC might pass for something that is tested in a different rule.

In writing this down, I think I figured out the solution to this too. This is fairly similar to what we did with inapplicable example. If a rule is inapplicable, that doesn't mean there isn't a failure of the mapped SC on that page. But in order for us to make the mapping work, the CG has added a requirement to the ACT rules format that says all inapplicable examples must satisfy the things in the accessibility requirements mapping. We can do a similar thing here: 

> All failed examples of rules used in a composite rule must fail the SCs of those composite rules.

Doing this will make the implementation mapping work, and I'm pretty sure we can do this without changes. Additionally, in order for us to get this implementation data, the CG will need to include all test cases of its atomic rules in the test cases list of a composite rule.

So, thoughts?

**Comment by tbostic32** [comment]
I think I am missing something. To me

> All failed examples of rules used in a composite rule must fail the SCs of those composite.

is saying that if the atomic rules used in the composite rule is failed then the SC is failed. That sounds like we would have a fail-to-fail relationship between the atomic rule and an SC (you fail the rule the SC fails)?

**Comment by WilcoFiers** [comment]
We have a solution for the composite rules we have today. We need to make sure atomic rules are surveyed and implemented before surveying the composite rules. The CG has an idea of how to do this.
